KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -417,6 +417,67 @@ public DeliveryClient(string projectId, string previewApiKey) : this(new Deliver
             return new ContentElement(response, elementCodename);
         }
 
+
+        /// <inheritdoc />
+        public async Task<JObject> GetTaxonomyJsonAsync(string codename)
+        {
+            if (codename == null)
+            {
+                throw new ArgumentNullException(nameof(codename), "The codename of a taxonomy group is not specified.");
+            }
+
+            if (codename == string.Empty)
+            {
+                throw new ArgumentException("The codename of a taxonomy group is not specified.", nameof(codename));
+            }
+
+            var endpointUrl = UrlBuilder.GetTaxonomyUrl(codename);
+
+            return await GetDeliverResponseAsync(endpointUrl);
+        }
+
+        /// <inheritdoc />
```

Comment:
I'm not a fan of duplicating code but in this case I'd vote for copying the XML docs from the interface. The reason for this is that VS doesn't support `<inheritdoc />`  in the intellisense.
Once it does we should switch to inherit doc and unify the style in the entire class.

Reply:
you can use r# to copy the comments. other than that, it looks fine